luskan/src/main/java/org/opencadc/luskan/AdqlQueryImpl.java

```diff
@@ -128,4 +135,8 @@ public String getSQL() {
         log.debug("SQL:\n" + sql);
         return sql;
     }
+
+    protected MultiValuedProperties getProperties() {
```

Comment:
One line method - would be more clear to just call the static method above.

Reply:
Made the protected one liner to allow unit test to have different luskan.properties configurations. Can toggle between a config for a storage site that should have the extra constraint injected in the query, and a global that should not.